src/sentry/runner/commands/config.py

```diff
@@ -44,9 +44,13 @@ def get(option, silent):
     # TODO(mattrobenolt): Add help to option keys
     # if key.help:
```

Comment:
Can we take out commented out code?

Reply:
yeah, will remove. Don't kjnow why it is there